### PR TITLE
Contact form with netlify

### PR DESCRIPTION
**Pull request by ethanWallace**
# Summary | Résumé

Contact form using Netlify forms

### CODE REVIEW

File: netlify/functions/submission-created.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    .then(data => {
+        if (!data.status_code) {
+            console.log(`Submitted to Notify`)
+        } else {
+            console.log(`Error - Status code: ${data.status_code}`);
+            for (const errors of data.errors) {
+                console.log(`Message: ${errors.message}`);
+            }
+        }
+    });
```
Comment (melaniebmn) [inline netlify/functions/submission-created.js:34]: Can we remove these console.log's?

Comment (melaniebmn) [inline netlify/functions/submission-created.js:34]: Can we remove these console.log's?

File: src/en/contact/contact.md [hunk 2]
```diff
@@ -1,12 +1,30 @@
...
   key: contactusEN
   title: Contact us
   locale: en
   order: 3
 translationKey: "contactus"
+contactForm: en
 ---
 
-# Contact us
+# Get in contact
```
Comment (melaniebmn) [inline src/en/contact/contact.md:13]: I think we discussed in the meeting yesterday to change this heading to "Contact us"

Comment (melaniebmn) [inline src/en/contact/contact.md:13]: I think we discussed in the meeting yesterday to change this heading to "Contact us"

File: src/en/contact/contact.md [hunk 3]
```diff
@@ -1,12 +1,30 @@
...
+  <gcds-input type="text" input-id="name" label="Full name"></gcds-input>
+  <gcds-input type="email" input-id="email" label="Email address"></gcds-input>
+  <gcds-textarea label="Message" textarea-id="message"></gcds-textarea>
+  <div hidden>
+    <gcds-input type="text" input-id="bot-field" label="bot"></gcds-input>
+  </div>
+  <gcds-button button-type="submit">
+    Send message
+  </gcds-button>
+</form>
```
Comment (melaniebmn) [inline src/en/contact/contact.md:30]: Can we replace this with the partial to make sure that both forms always get updated with future changes?

Comment (melaniebmn) [inline src/en/contact/contact.md:30]: Can we replace this with the partial to make sure that both forms always get updated with future changes?

File: src/_includes/partials/contactform.njk [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    <input type="email" name="email" aria-label="email"/>
+    <textarea name="message" aria-label="message"></textarea>
+    <input name="bot-field" aria-label="bot" hidden/>
+  </form>
+{% else %}
+  <form name="contactFR" data-netlify="true" netlify-honeypot="bot-field" action="/fr/contactez/merci" hidden>
+    <input type="text" name="name" aria-label="name"/>
+    <input type="email" name="email" aria-label="email"/>
+    <textarea name="message" aria-label="message"></textarea>
+    <input name="bot-field" aria-label="bot" hidden/>
```
Comment (melaniebmn) [inline src/_includes/partials/contactform.njk:13]: Aria-labels should be in french for the french form

Comment (melaniebmn) [inline src/_includes/partials/contactform.njk:13]: Aria-labels should be in french for the french form

**Review by melaniebmn — CHANGES_REQUESTED** [review]
Left a few comments

File: src/en/contact/contact.md (hunk 3 above)
Comment (ethanWallace) [reply src/en/contact/contact.md:30]: The form and partial form will need to remain separate so Netlify can process our form submissions.

File: netlify/functions/submission-created.js (hunk 1 above)
Comment (ethanWallace) [reply netlify/functions/submission-created.js:34]: I am in favour of leaving these in. This runs as a Netlify function and can be recorded in the error logs if anything happens on the Netlify side.

Comment (melaniebmn) [reply netlify/functions/submission-created.js:34]: Ok, sounds good. Let's leave them in.

**Review by melaniebmn — APPROVED** [review]
LGTM

**Pull request merged**